keras/activations.py

```diff
@@ -358,7 +358,8 @@ def gelu(x, approximate=False):
         if `approximate` is `False`.
 
     Reference:
-      - [Gaussian Error Linear Units (GELUs)](https://arxiv.org/abs/1606.08415)
+        - [Gaussian Error Linear Units (GELUs)](
+            https://arxiv.org/abs/1606.08415)
     """
```

Comment:
We keep these markdowns always on the same line and add a `# noqa: E501` after `"""`.